### PR TITLE
fix: add the article tag to wrapp the about the orgnization section

### LINKED ISSUE #1502: Improve the About the Organization section
# Description

The "About the Organization" section can be improved by wrapping the content, including its title, in a [article element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)

The code can be found in [`src/pages/organizations`](https://github.com/Greenstand/treetracker-web-map-client/blob/beta/src/pages/organizations/%5Borganizationid%5D.js)

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- For design guidelines checkout our [Figma design](https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap).
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by Jonath-z**
# Description

[comment]: # 'Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.'
This PR add the article tag to wrapp the about the organization section increasing the SEO

Fixes #1502 

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots

[comment]: # 'Please include screenshots of your changes if relevant.'

|       Before        |       After        |
| :-----------------: | :----------------: |
| "screenshot before" | "screenshot after" |

# How Has This Been Tested?

- [ ] Cypress integration
- [x] Cypress component tests
- [ ] Jest unit tests

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by RubenSmn — CHANGES_REQUESTED** [review]
We should aslo include the content in this article. Currently there is only the title, the content is the other lines below.

**Comment by Jonath-z** [comment]
@RubenSmn requested changes applied

**Review by RubenSmn — CHANGES_REQUESTED** [review]
Awesome looks really great, could you remove the article around the impact section? We're going to remove so no need.

**Comment by Jonath-z** [comment]
@RubenSmn changes added

**Comment by RubenSmn** [comment]
Thanks @Jonath-z 👍

**Comment by Jonath-z** [comment]
@RubenSmn no worries ... glad to help 😊

File: src/pages/organizations/[organizationid].js [hunk 1]
```diff
@@ -419,40 +419,48 @@ export default function Organization(props) {
...
-            />
-          </Typography>
-          <Typography variant="h4" sx={{ mt: { xs: 10, md: 16 } }}>
-            Mission
-          </Typography>
-          <Typography variant="body2" mt={7}>
-            <Box
-              component="span"
+          <article>
+            <Typography
```
Comment (dadiorchen) [inline src/pages/organizations/[organizationid].js:423]: @RubenSmn I think we can use this:
```
<Typography component="article" />
```
It's clear

Comment (dadiorchen) [inline src/pages/organizations/[organizationid].js:423]: @RubenSmn I think we can use this:
```
<Typography component="article" />
```
It's clear

Comment (RubenSmn) [reply src/pages/organizations/[organizationid].js:423]: The article here is only providing a landmark here. So I don't think we need the MUI `Typhography` component. However I just found a mistake that I overlooked when merging. I made [this pr to fix it](https://github.com/Greenstand/treetracker-web-map-client/pull/1506)

**Pull request merged**